### DISCUSSION TITLE
Support of Document Review

### DISCUSSION THREAD

**Discussion by mayank** [outline/outline#4106] [Ideas]
There is no document review process as of now. so every time document is updated, we need to manually check the contents and update it. And the document writer is unaware of changes done.

Suggestion: Needed a document review process that will submit the doc for review, and post approval and comment resolution it will get published.

**Comment by tommoor** [comment]
Fully aligned on the need for this, definitely on the roadmap

**Comment by michaelhannes** [comment]
Yes, nice.  I work for a company that is very motivated to have better knowledge sharing, but if you look at who is creating and editing documents it is always me.    

I would like to be able to hand off the document for review for other employees to basically "put the ball in their court," and to encourage collaboration.

**Comment by zesaver** [comment]
Subscribing for updates, for the moment this is the only blocker for using Outline in our company.
